s3/src/test/java/akka/stream/alpakka/s3/javadsl/DocSnippets.java

```diff
@@ -46,7 +47,8 @@ public String getRegion() {
                 Some.apply(proxy),
                 credentials,
                 regionProvider,
-                true
+                true,
+                Option.<String>none().asScala() //FIXME
```

Comment:
Still needs fix to allow a java Optional?

Reply:
I'm not sure I'm following. Are you suggesting we could simplify that last line?

Reply:
I saw the FIXME so my guess was you wanted something cleaner?

Reply:
Ah, no, sorry, I carried that over from master when I merged this branch with master. I'm not sure what the intent was... off-hand I'd guess they'd prefer to provide something smarter than a hardcoded none but weren't sure what.